main.py

```diff
@@ -115,6 +116,10 @@ def actuators(self, message, nick, pm):
             elif "!quote" in message.lower():
                 result = quote_day.quote_of_the_day()
                 self.send_msg(result, nick, pm)
+            elif "!wiki" in message.lower():
+                result = wiki_summary.scrape()
```

Comment:
It seems you forgot to add the 'topic' paramterer in the function call

Reply:
I'd be happy to merge when this is fixed :)
result = wiki_summary.scrape(message.split('!wiki', 1)[1])